common/arch/win32/rbaudio.cpp

```diff
@@ -48,91 +50,114 @@ void RBAExit()
     }
 }
 
-static int mci_HasMedia()
+static bool mci_HasMedia()
```

Comment:
Why is this return type changed?

Reply:
`bool` makes no sense, as the function can ever only logically return true/false.